docs/cli.rst

```diff
@@ -62,60 +63,124 @@ primary difference with simple `user` command is that password is being imported
   docker-compose run --rm admin flask mailu user-import myuser example.net '$6$51ebe0cb9f1dab48effa2a0ad8660cb489b445936b9ffd812a0b8f46bca66dd549fea530ce' 'SHA512-CRYPT'
 
 user-delete
-------------
+-----------
 
 .. code-block:: bash
 
   docker-compose exec admin flask mailu user-delete foo@example.net
 
+config-dump
+-----------
+
+The purpose of this command is to dump domain-, relay-, alias- and user-configuration to a YAML template.
+If you want to export non-hashed secrets you have to add the ``--secrets`` option.
+Only non-default options are dumped. If you want to dump all options use ``--full``.
+
+.. code-block:: bash
+
+  docker-compose exec admin flask mailu config-dump > mail-config.yml
+
 config-update
 -------------
 
-The sole purpose of this command is for importing users/aliases in bulk and synchronizing DB entries with external YAML template:
+The purpose of this command is for importing domain-, relay-, alias- and user-configuration in bulk and synchronizing DB entries with an external YAML template.
 
 .. code-block:: bash
 
-  cat mail-config.yml | docker-compose exec -T admin flask mailu config-update --delete-objects
+  docker exec -i $(docker-compose ps -q admin) flask mailu config-update -nvd < mail-config.yml
```

Comment:
Can you also document the short-hand parameters n, v , d?
You use -nvd in the example code without explaining what these parameters are.
n = --dry-run
v = --verbose
d = --delete-objects